src/Application/Blog/UseCase/Post/GetById/Command.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Blog\UseCase\Post\GetById;
+
+final class Command
```

Comment:
That should be query, not command, right?

Reply:
It has Command name in Command Bus pattern originally. 
It is not Command Bus pattern as full as could, at least now.

If you meant CQS pattern then it is not it, but looks similar.

Reply:
But there's no bus and I don't see it coming. Do we need an abstraction that we aren't going to use?

Reply:
> But there's no bus and I don't see it coming.

Yeap. But it can be added later.

What's yours suggestion?

Actual code base doesn't content any abstractions now. Is it clear and simple?